src/ert/config/analysis_module.py

```diff
@@ -41,7 +37,40 @@ class Config:
         validate_assignment = True
 
 
+InversionTypeES = Literal["exact", "subspace"]
+es_description = """
+    The type of inversion used in the algorithm. Every inversion method
+    scales the variables. The options are:
+
+    * `exact`:
+        Computes an exact inversion which uses a Cholesky factorization in the
+        case of symmetric, positive definite matrices.
+    * `subspace`:
+        This is an approximate solution. The approximation is that when
+        U, w, V.T = svd(D_delta) then we assume that U @ U.T = I.
+    """
+
+
+InversionTypeIES = Literal["direct", "subspace_exact", "subspace_projected"]
+ies_description = """
+    The type of inversion used in the algorithm. Every inversion method
+    scales the variables. The options are:
+
+    * `direct`:
+        Solve directly, which involves inverting a matrix
+        of shape (num_parameters, num_parameters).
```

Comment:
```suggestion
        of shape (num_observations, num_observations).
```

I have created a PR in `iterative_ensemble_smoother` to update the docstring:

https://github.com/equinor/iterative_ensemble_smoother/pull/207/files